core/core.lua

```diff
@@ -203,7 +204,10 @@ end
 local function GetColor(unit)
 	if unit then
 		local colorUnit = {}
-		if C.bar.marker and unit == "player" then
+		
+		if C.selfBar.enableCustomColor and UnitName("player") == UnitName(unit) then
+			return C.selfBar.customColor
+		elseif C.bar.marker and unit == "player" then
```

Comment:
this bit and the colorMarker code can be removed with the new custom logic.

Reply:
Removed